app/mixins/needs-discovered-hosts-ajax.js

```diff
@@ -6,12 +6,13 @@ export default Ember.Mixin.create({
     const token = Ember.$('meta[name="csrf-token"]').attr('content');
     let hypervisorIds = hypervisorHosts.sortBy('id').getEach('id');
     let hypervisorNames = hypervisorHosts.sortBy('id').getEach('name');
-    let arrayIdsNames = hypervisorIds.concat(hypervisorNames);
+    let hypervisorMacs = hypervisorHosts.sortBy('id').getEach('mac');
+    let arrayIdsNamesMacs = hypervisorIds.concat(hypervisorNames).concat(hypervisorMacs);
 
     return request({
       url: window.fusorServer + '/fusor/api/v21/deployments/' + deployment.get('id'),
       type: 'PUT',
-      data: JSON.stringify({'deployment': { 'discovered_host_ids_names': arrayIdsNames } }),
+      data: JSON.stringify({'deployment': { 'discovered_host_ids_names_macs': arrayIdsNamesMacs } }),
```

Comment:
fusor-server needs to be updated to handle this method `discovered_host_ids_names_macs=`